src/protocol/thread/thread.ts

```diff
@@ -36,14 +36,19 @@ import {
   responseBodyData,
   requestBodyData,
 } from "@recordreplay/protocol";
-import { client, log } from "../socket";
+import { client, log, addEventListener } from "../socket";
 import { defer, assert, EventEmitter, ArrayMap } from "../utils";
 import { MappedLocationCache } from "../mapped-location-cache";
 import { ValueFront } from "./value";
 import { Pause } from "./pause";
 import uniqueId from "lodash/uniqueId";
 import { repaint } from "protocol/graphics";
 
+declare global {
+  interface Window {
+    sessionMetrics: any;
```

Comment:
I know we probably won't be using it much, but it seems like we could probably at least make this like a `any[] | undefined` just to get *some* help from tsc?